Monika After Story/game/zz_pianokeys.rpy

```diff
@@ -807,6 +832,110 @@ init 1000 python in mas_piano_keys:
     # _pnm_<song name inital>_v#l#
     # v#l# -> verse #, line #
 
+
+###  HAPPY BIRTHDAY ###########################################################
+## full combo
+## w w e w t r
+## w w e w y t
+## w w o u t r e
+## i i u t y t
+
+    # happy birthday, piano note setup
+    _pnm_hb_v1l1 = PianoNoteMatch(
+        renpy.text.text.Text(
+            "Happy Birthday to you",
+            style="monika_credits_text"
+        ),
+        [
+            G4,
+            G4,
+            A4,
+            G4,
+            C5,
+            B4,
+        ],
+        postnotes=None,
+        express="1k",
+        postexpress="1a",
+        verse=0
+    )
+
+    _pnm_hb_v1l2 = PianoNoteMatch(
+        renpy.text.text.Text(
+            "Happy Birthday to you~",
+            style="monika_credits_text"
+        ),
+        [
+            G4,
+            G4,
+            A4,
+            G4,
+            D5,
+            C5,
+        ],
+        postnotes=None,
+        express="1k",
+        postexpress="1a",
+        verse=0
+    )
+    _pnm_hb_v1l3 = PianoNoteMatch(
+        renpy.text.text.Text(
+            "Happy Birthday Dear Friend",
```

Comment:
So I noticed you used "Friend" here, and I am aware that the usual `[player]` scheme doesn't work here.

In fact, it using dynamic text in the piano **really** doesn't work because all the note matches are created at init time, which means runtime changes (like changing name) wouldn't be properly reflected in piano.

"Friend", however, is 100% going to generate some controversy. 

I've heard this line said as `Happy Birthday, Happy birthday`, so I think changing it to that or some other way that doesn't require a name or pronoun would be better.

Later on, when I migrate these to text files, I can add something to dynamically apply the substitutes correctly so you can use the regular `[keyword]` replacement.